test/test_quantized.py

```diff
@@ -1322,6 +1323,129 @@ def test_qlinear_qnnpack(self, batch_size, input_channels, output_channels, use_
         np.testing.assert_equal(
             Y_q_ref2.int_repr().numpy(), Y_q.int_repr().numpy())
 
+    @given(batch_size=st.integers(1, 3),
+           input_channels_per_group=st.sampled_from([8, 16, 32]),
+           height=st.integers(10, 16),
+           width=st.integers(7, 14),
+           output_channels_per_group=st.sampled_from([8, 16, 32]),
+           groups=st.integers(1, 3),
+           kernel_h=st.integers(1, 7),
+           kernel_w=st.integers(1, 7),
+           stride_h=st.integers(1, 2),
+           stride_w=st.integers(1, 2),
+           pad_h=st.integers(0, 2),
+           pad_w=st.integers(0, 2),
+           dilation_h=st.integers(1, 1),
+           X_scale=st.floats(1.2, 1.6),
+           X_zp=st.integers(0, 4),
+           W_scale=st.floats(0.2, 1.6),
+           W_zp=st.integers(2, 5),
+           Y_scale=st.floats(4.2, 5.6),
+           Y_zp=st.integers(0, 4),
+           use_relu=st.booleans())
+    def test_qconv_qnnpack(
+            self,
+            batch_size,
+            input_channels_per_group,
+            height,
+            width,
+            output_channels_per_group,
+            groups,
+            kernel_h,
+            kernel_w,
+            stride_h,
+            stride_w,
+            pad_h,
+            pad_w,
+            dilation_h,
+            X_scale,
+            X_zp,
+            W_scale,
+            W_zp,
+            Y_scale,
+            Y_zp,
+            use_relu):
+        torch.backends.quantized.engine = torch.qnnpack
```

Comment:
since it's a global flag, you probably want to have a context manager to unset it. Otherwise it'd mess up other tests

Reply:
Do you have an example of how this can be done?

Reply:
Something like this:
https://github.com/pytorch/pytorch/blob/21ba320cd55bc6eecc280e18c1d8d1212a9737f0/test/jit_utils.py#L509-L515

In general, if the tests would be valid for both fbgemm and qnnpack, we can also make the entire Test class to set the engine in the beginning and instantiate it twice for each engine (potentially with some skips if some tests aren't supported)